.github/workflows/bots-automerge.yml

```diff
@@ -3,38 +3,32 @@ name: Bots auto-merge
 on: pull_request
 
 # This section adds write permissions to the secrets.GITHUB_TOKEN. Default is just read
-#permissions:
-  #contents: write
-  #pull-requests: write
+permissions:
```

Comment:
why is this needed?

Reply:
default permissions are mostly read-only for the github_token. This allows to set approvals on pull requests for GH actions and enable automerging.